<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2068</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>3000</width>
    <height>1500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color: #252525
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>471</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
	background-color: #616161;
	border-radius: 10px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="airportCode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	background-color: #f5f5f5;
	border: 0px;
	border-radius: 5px;
	color: #4f4f4f;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Origin Airport: (3 Letter Code)</string>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::LogicalMoveStyle</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="searchButton">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>60</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: #616970;
	border-radius: 5px;
	color: #f5f5f5;
}
QPushButton:pressed{
	border: 2px solid gray;
}

QPushButton:hover{
	background-color: #4b4a4a;
}</string>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>471</width>
       <height>261</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{
	background-color: #474747;
	border-radius: 10px;
	color: #f5f5f5;
	margin: 20px;

}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	background-color: #474747;
	color: #f5f5f5;
	border: 0px;
}</string>
      </property>
      <property name="text">
       <string>Flights Filter</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	background-color: #474747;
	color: #f5f5f5;
	border: 0px;
}</string>
      </property>
      <property name="text">
       <string>Month:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QSpinBox" name="maxDist">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>150</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QSpinBox{
	background-color: #f5f5f5;
	border-radius: 2px;
	color: #4f4f4f;
}</string>
      </property>
      <property name="minimum">
       <number>100</number>
      </property>
      <property name="maximum">
       <number>2000</number>
      </property>
      <property name="singleStep">
       <number>100</number>
      </property>
      <property name="value">
       <number>500</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_8">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>150</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	background-color: #474747;
	color: #f5f5f5;
	border: 0px;
}</string>
      </property>
      <property name="text">
       <string>Maximum Distance:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>80</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	background-color: #474747;
	color: #f5f5f5;
	border: 0px;
}</string>
      </property>
      <property name="text">
       <string>Year:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="monthSelect">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>80</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox{
	background-color: #f5f5f5;
	border: 2px;
	border-radius: 2px;
	color: #4f4f4f;
}</string>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="yearSelect">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>80</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox{
	background-color: #f5f5f5;
	border: 2px;
	border-radius: 2px;
	color: #4f4f4f;
}</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>360</y>
       <width>471</width>
       <height>241</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{
	background-color: #474747;
	border-radius: 10px;
	color: #f5f5f5;
	margin: 20px;

}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>411</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	background-color: #474747;
	color: #f5f5f5;
	border: 0px;
}</string>
      </property>
      <property name="text">
       <string>Search Algorithms</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="dfsButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #616970;
	border-radius: 5px;
	color: #f5f5f5;
}
QPushButton:pressed{
	border: 2px solid gray;
}

QPushButton:hover{
	background-color: #4b4a4a;
}</string>
      </property>
      <property name="text">
       <string>DFS</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bfsButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>90</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #616970;
	border-radius: 5px;
	color: #f5f5f5;
}
QPushButton:pressed{
	border: 2px solid gray;
}

QPushButton:hover{
	background-color: #4b4a4a;
}</string>
      </property>
      <property name="text">
       <string>BFS</string>
      </property>
     </widget>
     <widget class="QPushButton" name="dijkstraButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #616970;
	border-radius: 5px;
	color: #f5f5f5;
}
QPushButton:pressed{
	border: 2px solid gray;
}

QPushButton:hover{
	background-color: #4b4a4a;
}</string>
      </property>
      <property name="text">
       <string>dijkstra</string>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	background-color: #616161;
	color: #f5f5f5;
	border: 0px;
}</string>
     </property>
     <property name="text">
      <string>Route Finder</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>110</y>
      <width>641</width>
      <height>521</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2068</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRoute_Finder">
    <property name="title">
     <string>Route Finder</string>
    </property>
   </widget>
   <addaction name="menuRoute_Finder"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
